plugins/grpc/node/v1.11.3/buf.plugin.yaml

```diff
@@ -0,0 +1,21 @@
+version: v1
+name: buf.build/grpc/node
+plugin_version: v1.11.3
+source_url: https://github.com/grpc/grpc-node
+description: Generates NPM client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/js:v3.21.2
+output_languages:
+  - javascript
+registry:
+  npm:
+    import_style: commonjs
+    deps:
+    - package: '@grpc/grpc-js'
+      version: '^1.7.0'
+    - package: 'google-protobuf'
+      version: '^3.21.2'
+  opts:
+  - grpc_js
```

Comment:
There was actually a bug in the previous version: https://github.com/bufbuild/plugins/pull/153/files#diff-7c1a6156c206b8fc008eac7d05d4fea580630a18b83a9c63d3a1b8f62b539c3aL7